advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>Branding nit: <code>Catboost</code> -&gt; <code>CatBoost</code>	(2-3 occurrences)</p><p> </p><p>Typo: <code>demostrate</code> -&gt; <code>demonstrate </code>(2 occurrences)</p><p> </p><p>Grammar: <code>show case</code> -&gt; <code>showcase</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3508/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thank you for catching this , have fixed in the latest push</p>

Reply:
fixed